src/Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoGridHelpers.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoGridHelpers.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"  // IWYU pragma: keep
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"  // IWYU pragma: keep
+#include "Domain/Side.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace SlopeLimiters {
+namespace Weno_detail {
+
+template <size_t VolumeDim>
+std::array<DataVector, VolumeDim> neighbor_grid_points_in_local_logical_coords(
+    const Mesh<VolumeDim>& neighbor_mesh,
+    const Direction<VolumeDim>& direction_to_neighbor) noexcept {
+  std::array<DataVector, VolumeDim> result{{{}}};
+  for (size_t d = 0; d < VolumeDim; ++d) {
+    if (d == direction_to_neighbor.dimension()) {
+      gsl::at(result, d) =
+          get<0>(logical_coordinates(neighbor_mesh.slice_through(d))) +
+          (direction_to_neighbor.side() == Side::Upper ? 2.0 : -2.0);
```

Comment:
A comment stating that `2.0` comes from or making it a variable name `logical_element_width` would be helpful.